meta-lmp-bsp/recipes-bsp/u-boot/u-boot-ostree-scr/am62xx-evm/uEnv.txt.in

```diff
@@ -0,0 +1,6 @@
+bootcmd_dtb=run findfdt; setenv loadaddr 0x90000000
+bootcmd_load_env=load mmc ${mmcdev}:2 ${loadaddr} /boot/loader/uEnv.txt
+bootcmd_run_env=env import -t ${loadaddr} ${filesize}
+bootcmd_load_k=load mmc ${mmcdev}:2 ${loadaddr} ${kernel_image}
+bootcmd_run=bootm ${loadaddr}#conf@@FIT_NODE_SEPARATOR@@ti_${fdtfile}
```

Comment:
Since this is a new target, and with a recent enough u-boot, you can drop "@@FIT_NODE_SEPARATOR@@" and use "-" instead directly.